workspaces/frontend/src/shared/style/MUI-theme.scss

```
@@ -267,6 +267,10 @@
   resize: none;
   --pf-v6-c-form-control--PaddingBlockStart: var(--mui-spacing-16px);
   --pf-v6-c-form-control--PaddingBlockEnd: var(--mui-spacing-16px);
+  #text-file-simple-filename {
+    --pf-v6-c-form-control--PaddingBlockStart: var(--mui-spacing-8px);
+    --pf-v6-c-form-control--PaddingBlockEnd: var(--mui-spacing-8px);;
+  }
 }
 
 
```

In step 2, the vertical gap between input fields in the form currently appears too large. To fix this, add this to line 105 of MUI-theme.scss:

```suggestion
    --pf-t--global--spacer--gap--group-to-group--vertical--default: var(--pf-t--global--spacer--sm); 
```